index.html

```diff
@@ -60,6 +60,157 @@ <h3 class="content-2">I am a Software Developer</h3>
               </ul>
             </article>  
         </section>
+        
+        <section class="second-section">
+               <div class="page-title">
+                <h1 class="section-title">My Recent Works</h1>
+               </div>
+             
+              <div class="feature-wrapper">
+                <div><img src="./images/feature.png" alt=""></div>
+                <h2 class="project-title">Multi-Post Stories</h2>
+                <p class="project-discription">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's 
+                  standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="technologies">
+                  <li>css</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+                <button class="button btn">See Project</button>
+              </div>
```

Comment:
- Good job on the design but please stay as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Kindly the margin that you gave to the button.

Here you have a screenshot of the button:

![image](https://user-images.githubusercontent.com/74506933/154390164-bdf7e502-9155-42e5-8f54-5c492e0d7a04.png)